plugin/kubernetes/root.go

```diff
@@ -21,7 +21,6 @@ import (
 // Root of the Kubernetes plugin
 type Root struct {
 	plugin.EntryBase
-	contexts []plugin.Entry
```

Comment:
This is the one change I'm most nervous about. I tried to keep it around, but it wasn't really necessary so any attempts seemed weird and made the code needlessly confusing.

Reply:
I agree we don't need it anymore. Thanks for the PR!